tests/docker/test-docker-compose.yml

```diff
@@ -46,12 +46,10 @@ services:
       - ./.env
     depends_on:
       - bridge0
```

Comment:
Should geth depend on metro instead of the bridge? It seems odd to depend on the Celestia bridge, but it needs to read the metro keys.